src/app/views/query-response/QueryResponse.tsx

```diff
@@ -101,16 +104,48 @@ const QueryResponse = (props: IQueryResponseProps) => {
     }
   };
 
+  const selectContextItem = (e: any, item: any, link: any) => {
```

Comment:
Consider using the appropriate type instead of **any**

Reply:
Thank you. 
Fixed!